integration_tests/dataproc_test_case.py

```diff
@@ -199,6 +199,14 @@ def tearDown(self):
     def getClusterName(self):
         return self.name
 
+    @staticmethod
+    def getBaseOS():
+        # Get the base operating system: '1.5-debian10' -> 'debian'
+        # Dataproc default "preview" version is Debian, so "debian" 
+        # is returned if just "preview" is provided for the image version.
+        return "debian" if FLAGS.image_version == "preview" else \
```

Comment:
Maybe let's make it more generic for all images, i.e. `1.3`, `1.4`, `1.5`?

Reply:
Ack, fixed